Taskfile.yml

```diff
@@ -43,7 +43,7 @@ tasks:
     - task: generator:ci
     - task: controller:generate-crds # required here to prevent it running twice in controller:ci, remove once go-task dedupes dependencies
     - task: controller:ci
-    - task: crossplane:ci
+    #- task: crossplane:ci
```

Comment:
Need to fix SQL not being able to be flattened.

Reply:
Before you merge this PR or after?

Reply:
It feels bad to comment out things to pass CI ;)

Reply:
seems to work again, perhaps #1556 fixed it